packages/grafana-data/src/types/logs.ts

```diff
@@ -163,6 +163,26 @@ export enum SupplementaryQueryType {
   LogsSample = 'LogsSample',
 }
 
+/**
+ * @internal
+ */
+export type SupplementaryQueryOptions = LogsVolumeOption | LogsSampleOptions;
+
+/**
+ * @internal
+ */
+export type LogsVolumeOption = {
+  type: SupplementaryQueryType.LogsVolume;
+};
+
+/**
+ * @internal
+ */
+export type LogsSampleOptions = {
```

Comment:
Why did you mark these as internal? I think it's the first time I see this annotation used.